Gitweb:     http://git.kernel.org/git/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commit;h=a229fc61ef0ee3c30fd193beee0eeb87410227f1
Commit:     a229fc61ef0ee3c30fd193beee0eeb87410227f1
Parent:     cec0707e40ae25794b5a2de7b7f03c51961f80d9
Author:     Boaz Harrosh <bharrosh@panasas.com>
AuthorDate: Mon Jan 19 10:37:38 2009 +0100
Committer:  Jens Axboe <jens.axboe@oracle.com>
CommitDate: Fri Jan 30 12:34:37 2009 +0100

    include/linux: Add bsg.h to the Kernel exported headers
    
    bsg.h in current form is perfectly suitable for user-mode
    consumption. It is needed together with scsi/sg.h for applications
    that want to interface with the bsg driver.
    
    Currently the few projects that use it would copy it over into
    the projects. But that is not acceptable for projects that need
    to provide source and devel packages for distros.
    
    This should also be submitted to stable 2.6.28 and 2.6.27 since bsg had
    a stable API since these Kernels and distro users will need the header
    for these kernels a swell
    
    Signed-off-by: Boaz Harrosh <bharrosh@panasas.com>
    Acked-by: FUJITA Tomonori <fujita.tomonori@lab.ntt.co.jp>
    CC: stable@kernel.org
    Signed-off-by: Jens Axboe <jens.axboe@oracle.com>

 include/linux/Kbuild |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/linux/Kbuild b/include/linux/Kbuild
index 12e9a29..2124c06 100644
--- a/include/linux/Kbuild
+++ b/include/linux/Kbuild
@@ -41,6 +41,7 @@ header-y += baycom.h
 header-y += bfs_fs.h
 header-y += blkpg.h
 header-y += bpqether.h
+header-y += bsg.h
 header-y += can.h
 header-y += cdk.h
 header-y += chio.h
